src/Components/TaskManagerComponent/TaskManagerComponent/WebApi.ts

```diff
@@ -13,6 +13,13 @@ export interface IWebApi extends ComponentFramework.WebApi {
 }
 
 export class WebApi implements IWebApi {
+    private static readonly API_RELATIVEPREFIX: string = "api/data/v9.2";
```

Comment:
Nitpick: It might make more sense to create a `ApiBaseUrl` property that returns clientUrl + API_RELATIVEPREFIX instead of repeating API_RELATIVEPREFIX throughout.